### ISSUE TITLE
Neon blank for a couple of files (unifying factor yet unknown)

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Neon#794]
When I upload them to Neon, the files of Einsie folios 262r and 266r look like this:
<img width="1440" alt="Blank neon" src="https://user-images.githubusercontent.com/83373378/163390231-b75405c3-a40a-4017-a42c-e549284c8c7e.png">
No matter how long I wait, the page stays blank and the MEI status stays at "unknown."
I tried the following things, but none of them worked:
- refreshing the page
- reuploading the file to Neon
- reprocessing the folio through the complete OMR and uploading that new file
- @annamorphism also tried uploading the file and got the same result

Looking at the MEI files, I can't tell what's wrong... Here they are:
[262r.mei.zip](https://github.com/DDMAL/Neon/files/8489134/262r.mei.zip)
[266r.mei.zip](https://github.com/DDMAL/Neon/files/8489136/266r.mei.zip)

So far this has only happened twice, but if ever we come across this again, we'll upload the files here.

**Comment by martha-thomae** [comment]
I opened the files in Oxygen and saw that both of them have some `<divLine>` elements within `<syllable>`, I don't know if this is common and also happens on other files that do work on Neon. But if this is not common, that might be the issue. There is one example in `266r.mei` with a huge `<syllable>` in lines 2314--2403 that has more than one `<divLine>` within.

**Comment by JoyfulGen** [comment]
@yinanazhou implemented a new feature that allows the user to add a `divLine` to a syllable and it's been working fine so far, so I'm not sure that's the problem. Looking at that giant syllable, however, it has at least one `<sb>` inside it, which is very strange. 
I think this issue has to do with https://github.com/DDMAL/Neon/issues/676, where we discussed the `<sb>` appearing within a syllable, rather than the syllable being toggled around it. It's strange that this is still happening, but it could very well be the source of the problem.

**Comment by yinanazhou** [comment]
@JoyfulGen, are these files the direct output from the OMR process?

**Comment by yinanazhou** [comment]
This issue is caused by the automatic check of `@precedes` and `@follows` when uploading files in Neon. Neon will automatically check whether `@precedes` and `@follows` are correct when `sb`, `custos` or `clef` are found inside `syllable`. However, in folio 262r, `sb` is found in `syllable` but the syllable has no `@precedes` or `@follows`, thus causing an error in uploading files, and you see a blank page. 

I've realized that the upload function should not change the file. I will delete the automatic check, and add a message in Neon to show where is wrong with the folio. 

However, no `sb` should be inside `syllable`. I've already changed the settings in Neon. @precedes and @follows do not mess up with the position of `sb`, `clef`, or `custos`. If these files are the direct output from the OMR process, we need to check the MEI-encoding job in Rodan.
